sdks/go/node/api/listen.go

```diff
@@ -73,31 +45,23 @@ func (hd _httpListener) listen(
 
 	router.PathPrefix("/").Handler(http.FileServer(http.FS(buildFS)))
 
-	httpServer := http.Server{
+	apiServer := http.Server{
 		Addr:    address,
 		Handler: handlers.CORS()(router),
 	}
 
-	// catch signals to ensure shutdown properly happens
-	done := make(chan os.Signal, 1)
```

Comment:
pulled this out of the listener since we have multiple listeners now (API and DNS); we want to capture signal above them and handle via context.